website/blog/2023-09-10-databend-weekly-110.md

```diff
@@ -0,0 +1,160 @@
+
+Databend provides Python Binding, allowing you to access Databend without deploying a instance. Its DataFrames can also be easily converted to Polars and Pandas formats, making it convenient for integration with data science tools.
+
+Just execute the following command to install:
+
+```bash
+pip install databend
+```
+
+The code below demonstrates how to use Databend Python Binding in Jupyter Notebook and plot a bar chart using `matplotlib`.
+
+```python
+# Create a table in DataBend
+ctx.sql("CREATE TABLE IF NOT EXISTS user (created_at Date, count Int32)")
+
+# Create a table in DataBend
+ctx.sql("CREATE TABLE IF NOT EXISTS user (created_at Date, count Int32)")
+
+# Insert multiple rows of data into the table
+ctx.sql("INSERT INTO user VALUES ('2022-04-01', 5), ('2022-04-01', 3), ('2022-04-03', 4), ('2022-04-03', 1), ('2022-04-04', 10)")
+
+# Execute a query
+result = ctx.sql("SELECT created_at as date, count(*) as count FROM user GROUP BY created_at")
+
+# Display the query result
+result.show()
+
+# Import libraries for data visualization
+import matplotlib.pyplot as plt
+
+# Convert the query result to a Pandas DataFrame
+df = result.to_pandas()
+
+# Create a bar chart to visualize the data
+df.plot.bar(x='date', y='count')
+plt.show()
+```
+
+![](https://databend.rs/assets/images/localhost_8888_notebooks_Untitled.ipynb-3992998c3577fa62ff83a36bcee494f8.png)
+
+If you are interested in learning more, please check out the resources listed below:
+
+- [Docs | Tutorial-3: Integrate with Jupyter Notebook with Python Binding Library](https://databend.rs/doc/visualize/jupyter#tutorial-3-integrate-with-jupyter-notebook-with-python-binding-library)
+
+## Highlights
+
+We have also made these improvements to Databend that we hope you will find helpful:
+
+- Added initial support for ownership model.
+- Added support for Hash Join spill.
+- Columns in Databend are now nullable by default. Please refer to the documentation [Docs | NULL Values and NOT NULL Constraint](https://databend.rs/doc/sql-reference/data-types/#null-values-and-not-null-constraint) for more details.
+- Read the document [Docs | databend-local](https://databend.rs/doc/sql-clients/databend-local) to master Databend Local mode.
+
+## What's Up Next
+
+We're always open to cutting-edge technologies and innovative ideas. You're more than welcome to join the community and bring them to Databend.
+
+### Creating UDFs with JavaScript
+
+[PR #12729 | feat: implement udf server in databend](https://github.com/datafuselabs/databend/pull/12729) is expected to be merged this week. This means that Databend will soon support creating user-defined functions with Python.
+
+```SQL
+CREATE FUNCTION [IF NOT EXISTS] <udf_name> (<arg_type>, ...) RETURNS <return_type> LANGUAGE <language> HANDLER=<handler> ADDRESS=<udf_server_address>
+```
+
+We expect to provide support for various language UDFs based on this PR, and JavaScript may be one of the options worth trying.
```

Comment:
We will provide support for UDFs created in various languages, with JavaScript being the next one in line.